src/main/java/com/ita/if103java/ims/dao/impl/UserDaoImpl.java

```diff
@@ -59,7 +56,7 @@ public User create(User user) {
                     "Autogenerated key is null")));
             return user;
         } catch (DataAccessException e) {
-            throw crudException(e.getMessage(), "create", "id = " + user.getId());
+            throw crudException(e.toString(), "create", "id = " + user.getId());
```

Comment:
`throw crudException(e, "create", "id = " + user.getId());`

Reply:
Will show you on Friday what I want you do

Reply:
But I think in this case 
throw crudException(e, "create", "id = " + user.getId());
and 
throw crudException(e.toString(), "create", "id = " + user.getId());
will give us the same or am I wrong?

Reply:
I don't know, please check and let me know.

But what I want to have at the  end in the crudException()
```
String msg = "CRUDException exception. Operation:(" + operation + ") user (" + attribute + ") exception.;
LOGGER.error(msg, e);
```

If the result is the same, let it be, but passing exception object into logger is better solution